protocols/v2/channels-sv2/src/client/extended.rs

```diff
@@ -1,4 +1,20 @@
-//! Mining Client abstraction over the state of a Sv2 Extended Channel
+//! # Sv2 Extended Channel - Mining Client Abstraction
+//!
+//! This module provides an abstraction over the state of an [Sv2](https://stratumprotocol.org/specification)
+//! **Extended Channel** within a mining client.
+//!
+//! Extended Channels are primarily intended for use by mining proxies to enable more efficient
+//! management and distribution of the hashing space among downstream clients.
+//!
+//! ## Key Concepts
+//!
+//! An Extended Channel is characterized by:
+//!
+//! - **`extranonce_prefix`**: A prefix defined by the upstream pool, unique to each channel.
+//! - **`extranonce_size`**: The number of bytes available for use by the local proxy or
+//!   distribution to downstream clients for nonce rolling.
```

Comment:
hmm it's characterized by more things?

but also, I guess we don't need a lengthy description on the module header
```suggestion
//! ## Key Concepts
//!
//! An Extended Channel is characterized by:
//!
//! - **`extranonce_prefix`**: A prefix defined by the upstream pool, unique to each channel.
//! - **`extranonce_size`**: The number of bytes available for use by the local proxy or
//!   distribution to downstream clients for nonce rolling.
```